backend/tracim_backend/lib/core/event.py

```diff
@@ -60,6 +60,8 @@
 
 RQ_QUEUE_NAME = "event"
 
+_PENDING_EVENTS = "pending_events"
```

Comment:
This seems not used or do i miss something ?

Reply:
Ah yes, I forgot this.